src/doxygen.cpp

```diff
@@ -9362,7 +9362,7 @@ static void copyStyleSheet()
   }
 }
 
-static void copyLogo(const QCString &outputOption)
+static void copyLogo(const QCString &outputOption, const QCString docFormat)
```

Comment:
Any reason to pass a "magic" string value here instead of a boolean/enum indicating whether this item needs to be added to the index?

Reply:
A "magic" string is not the best idea.
I think I couldn't find a suitable enum for it. 
The closest what I see, now, is the enun Type in the class DocImage (in docparser.h), I don't think this is the best choice either (the copyLogo is about an image but not related to DocImage so taking it is not good). Probably best is to define an new enum class